src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/samples/Sample7_ConfiguringPipeline.cs

```diff
@@ -54,19 +54,19 @@ public async Task ConfiguringPipeline()
 
         class AddHeaderPolicy : HttpPipelinePolicy
         {
-            public override async Task ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPipelinePolicy> pipeline)
+            public override async Task ProcessAsync(HttpPipelineContext pipelineContext, ReadOnlyMemory<HttpPipelinePolicy> pipeline)
```

Comment:
Why don't we keep the mane as it was? We would have message, request, response.

Reply:
I don't have a very strong opinion about this but it seemed to me like request and response are messages, context is more like an object that holds things required by pipeline to execute.

Also somewhat aligns with java
https://github.com/Azure/autorest-clientruntime-for-java/blob/v3/azure-common/src/main/java/com/azure/common/http/HttpPipelineCallContext.java

Reply:
I would prefer to stick with the "Message" terminology.

Reply:
I'll change back to the message. 

> Moreover these three types should either all start with HttpPipeline, or all with Http, or all with Pipeline. As opposed to HttpPipelineContext, but PipelineRequest or HttpRequest.

They all do, `PipelineRequest` is internal to `HttpClientTransport`

Reply:
Done.